Tests/NetworkRequestTests.swift

```diff
@@ -104,4 +104,61 @@ class NetworkRequestTests: XCTestCase {
         XCTAssertEqual(urlRequest.cachePolicy, cachePolicy, file: file, line: line)
         XCTAssertEqual(urlRequest.timeoutInterval, timeout, file: file, line: line)
     }
+    
+    func test_NetworkRequest_InitEmptyRequest() {
+        let response = EmptyResponse()
+        XCTAssert(true)
+    }
+    
+    struct CachePolicyAndTimeOutRequest: NetworkRequest {
+        typealias ResponseType = EmptyResponse
+        typealias ErrorType = AnyError
+        
+        var method: HTTP.Method = NetworkRequestTests.defaultRequestMethod
+        var url = NetworkRequestTests.defaultURL
+        var queryParameters: [URLQueryItem]?
+        var headers: [HTTP.HeaderKey: HTTP.HeaderValue]?
+        var body: Data?
+    }
+ 
+    func test_NetworkRequest_GetCachePolicy() {
```

Comment:
I might tweak the name of these slightly as well since you're really testing that a `NetworkRequest` created without an explicit cache policy and timeout return the default cache policy and timeout, right?
```
func test_NetworkRequestWithoutExplicitCachePolicyAndTimeout_ReturnsDefaultCachePolicyAndTimeout()
```

Reply:
Done